templates/policy-template/src/policies/hello-rego.rego.tmpl

```diff
@@ -7,14 +7,14 @@ default allowed = false
 default visible = false
 default enabled = false
 
-allowed {
+allowed if {
```

Comment:
How does this work when I create a `rego.v0` policy? 

I would expect when using `rego-version=rego.v0` we generate a policy using the existing template (no if) and when using rego.v1 you get the new syntax.

Should we, therefore, have two versioned templates of a parameterized single one that can handle this?

Reply:
I'll update the template command to accept the rego version parameter and prepare different templates for rego v0 and rego v1